lib/core/src/server/templates/index.html.ejs

```diff
@@ -7,17 +7,23 @@
     <title>Storybook</title>
     <link rel="shortcut icon" href="favicon.ico?v=1" />
 
+    <% if (options.headHtmlSnippet) { %>
+      <%- options.headHtmlSnippet %>
+    <% } %>
+      
     <% if (options.links) { %>
```

Comment:
Is there still a point on having `options.links` when we don't (well, can't) use them?

Reply:
Well not really no. we do not expose a way to set this.